docs/Installation/Installation.md

```diff
@@ -131,6 +131,48 @@ Notes:
           -D CHARM_ROOT=/work/charm/multicore-linux64-clang SPECTRE_ROOT
 ```
 
+## Using Singularity to obtain a SpECTRE environment
+
+[Singularity](http://singularity.lbl.gov/index.html) is a container alternative
+to Docker with better security and nicer integration.
+
+To use Singularity you must:
+
+1. Build [Singularity](http://singularity.lbl.gov/index.html) and add it to your
+   `$PATH`
+2. `cd` to the directory where you want to store the SpECTRE Singularity image,
+   source, and build directories, let's call it WORKDIR, which must be somewhere
+   in your home directory. Clone SpECTRE into `WORKDIR/SPECTRE_ROOT`.
+3. Run `singularity build spectre.img
```

Comment:
When I do the singularity build command it says

ERROR UNAUTHORIZED: authentication required
ERROR Check existence, naming, and permissions